### DISCUSSION TITLE
Sync between devices?

### DISCUSSION THREAD

**Discussion by sagraham** [Sequel-Ace/Sequel-Ace#1188] [Ideas]
Is there a way that we could have some kind of Sync between devices? I'd really like it if the favourites I create on my iMac were available on my MacBook Air and vice versa.

I know there is export, which is excellent, but I'd love something automatically working in the background, being two-way and ideally being encrypted so passwords could also be synced.

If a full, encrypted two-way sync isn't likely, is it possible there is a way of storing the relevant files in an iCloud Drive and creating symbolic links to them on each device?

Totally appreciate this is a difficult, hard problem and may be beyond the current roadmap of the app.

**Comment by Jason-Morcos** [comment]
That would certainly be very cool, but probably isn't that trivial to implement. Maybe related to #53

**Reply by sagraham** [reply]
Ah, missed that thread. Thanks.
